agent/src/beerocks/monitor/monitor_rssi.cpp

```diff
@@ -38,8 +38,6 @@ void monitor_rssi::stop()
         close(arp_socket);
         arp_socket = -1;
     }
-    mon_db       = nullptr;
```

Comment:
Why did you delete these lines? It is not clear from the commit message.

Reply:
nullptr assignment is not needed and not contributing anything. When the monitor calls to `mon_rssi.start()` and fails, it calls to `stop()` BUT it continue calling to `process()` on each work cycle, which uses these pointers.